config/install.sh

```diff
@@ -29,17 +29,16 @@ curl -s "https://raw.githubusercontent.com/equinor/amplify-components/main/confi
 
 printf -- "Downloading setupTests folder...\n"
```

Comment:
Nitpicky but this should say "Going into test-utils folder" or something like that